node/actors/bft/src/chonky_bft/commit.rs

```diff
@@ -1,43 +1,58 @@
-//! Handler of a ReplicaCommit message.
```

Comment:
"misc" doesn't mean anything. If you cannot find a good place for this logic, then just move it to the parent module.